src/sentry/signals.py

```diff
@@ -72,22 +78,22 @@ def wrapped(func):
             wrapped.__doc__ = receiver.__doc__
         return wrapped(receiver)
 
-    def send_robust(self, sender, **named):
+    def send_robust(self, sender, **named) -> List[Tuple[Receiver, Union[Exception, Any]]]:
         """
         A reimplementation of send_robust which logs failures, thus recovering stacktraces.
         """
-        responses = []
+        responses: List[Tuple[Receiver, Union[Exception, Any]]] = []
         if not self.receivers or self.sender_receivers_cache.get(sender) is NO_RECEIVERS:
             return responses
 
         # Call each receiver with whatever arguments it can accept.
         # Return a list of tuple pairs [(receiver, response), ... ].
-        for receiver in self._live_receivers(sender):
+        for receiver in self._live_receivers(sender):  # type: ignore[attr-defined]
```

Comment:
we can probably send a patch to django-stubs to add this -- even though it's private